runtime/oti/j9consts.h

```diff
@@ -276,7 +276,6 @@ extern "C" {
 #define J9_RUNTIME_ARGENCODING_UNICODE 0x2000
 #define J9_RUNTIME_ARGENCODING_LATIN 0x4000
 #define J9_RUNTIME_ARGENCODING_UTF8 0x8000
-#define J9_RUNTIME_DENY_ILLEGAL_ACCESS 0x10000
```

Comment:
Please replace this with J9_RUNTIME_UNUSED_0x10000 instead of just removing the line.

Reply:
I started to do that, but there are four unused bits: would you like a macro like this for each?
```
#define J9_RUNTIME_UNUSED_0x10000 0x10000
#define J9_RUNTIME_UNUSED_0x20000 0x20000
#define J9_RUNTIME_UNUSED_0x40000 0x40000
#define J9_RUNTIME_UNUSED_0x80000 0x80000
```

Reply:
Please - I fixed one today that was just called FLAG1, which would be especially hard to maintain.

Reply:
Placeholder macros added.